client/lib/preferences-helper/preference-list.js

```diff
@@ -17,31 +16,42 @@ import Preference from './preference';
 class PreferenceList extends Component {
 	render() {
 		const { preferences, translate } = this.props;
+
+		const sortedPreferenceItems = preferences
+			? Object.keys( preferences )
+					.sort()
+					.map( ( preferenceName ) => (
+						<Preference
+							key={ preferenceName }
+							name={ preferenceName }
+							value={ preferences[ preferenceName ] }
+						/>
+					) )
+			: [];
+
 		return (
-			<div>
+			<>
 				<QueryPreferences />
-				<a
-					href={ '/devdocs/client/state/preferences/README.md' }
-					title={ translate( 'Preferences' ) }
-				>
-					{ translate( 'Preferences' ) }
-				</a>
+				<a href={ '/devdocs/client/state/preferences/README.md' }>{ translate( 'Preferences' ) }</a>
 				<Card className="preferences-helper__current-preferences">
-					{ ! isEmpty( preferences ) ? (
-						Object.keys( preferences ).map( ( preferenceName ) => (
-							<Preference
-								key={ preferenceName }
-								name={ preferenceName }
-								value={ preferences[ preferenceName ] }
-							/>
-						) )
+					{ sortedPreferenceItems.length > 0 ? (
+						<table className={ 'preferences-helper__preferences-table' }>
+							<thead>
+								<tr>
+									<th className="preferences-helper__header-unset">{ translate( 'Delete' ) }</th>
+									<th className="preferences-helper__header-name">{ translate( 'Name' ) }</th>
+									<th className="preferences-helper__header-value">{ translate( 'Value(s)' ) }</th>
+								</tr>
+							</thead>
+							<tbody>{ sortedPreferenceItems }</tbody>
+						</table>
 					) : (
-						<h5 className="preferences-helper__preference-header">
-							{ translate( 'No Preferences' ) }
+						<h5 className="preferences-helper__no-preferences">
+							{ translate( 'No preferences are currently set' ) }
```

Comment:
This could be a bit incorrect in the instance when the preferences were not successfully loaded for some reason.